var/spack/repos/builtin/packages/r-genomicranges/package.py

```diff
@@ -40,14 +40,21 @@ class RGenomicranges(RPackage):
     homepage = "https://bioconductor.org/packages/GenomicRanges/"
     git      = "https://git.bioconductor.org/packages/GenomicRanges.git"
 
+    version('1.32.6', commit='31426be0fd6b76f7858971dc45aaf6e6d1dbac4e')
     version('1.30.3', commit='e99979054bc50ed8c0109bc54563036c1b368997')
     version('1.28.6', commit='197472d618f3ed04c795dc6ed435500c29619563')
 
     depends_on('r-biocgenerics@0.21.2:', type=('build', 'run'))
+    depends_on('r-biocgenerics@0.25.3:', when='@1.32.6', type=('build', 'run'))
     depends_on('r-s4vectors@0.9.47:', type=('build', 'run'))
-    depends_on('r-iranges@2.9.11:', type=('build', 'run'), when='@1.28.6')
-    depends_on('r-iranges@2.11.16:', type=('build', 'run'), when='@1.30.3')
-    depends_on('r-genomeinfodb@1.11.5:', type=('build', 'run'), when='@1.28.6')
-    depends_on('r-genomeinfodb@1.13.1:', type=('build', 'run'), when='@1.30.3')
+    depends_on('r-s4vectors@0.17.32:', when='@1.32.6', type=('build', 'run'))
+    depends_on('r-iranges@2.9.11:', when='@1.28.6', type=('build', 'run'))
+    depends_on('r-iranges@2.11.16:', when='@1.30.3', type=('build', 'run'))
+    depends_on('r-iranges@2.14.4:', when='@1.32.6', type=('build', 'run'))
+    depends_on('r-genomeinfodb@1.11.5:', when='@1.28.6', type=('build', 'run'))
```

Comment:
these will all go to the newest `r-genomeinfodb`, just checking that this is actually what you meant. Also see `r-iranges`